### ISSUE TITLE
.NET MVC

### ISSUE THREAD

**Issue by aliselmes** [aliselmes/recipe-tracker#2]

**Comment by aliselmes** [comment]
Quick question: when I am migrating my existing "Recipe Tracker" pages into the .NET core pages I need to just replace what is in the core pages? For example, in the "site.css" file I can just delete everything that is in there and replace it with my css from my recipe tracker pages? Or in the "_Layout.cshtml" page I can replace the footer that is currently in there with the footer from my recipe tracker pages?

**Comment by Nhawdge** [comment]
Exactly. replace the demo stuff with your actual code. Once the demo stuff is all gone, it'll start to feel more like home. Feel free to use the demo files for a general guideline of organizational stuff though.

**Comment by aliselmes** [comment]
Hi John, quick question about how to structure the recipe tracker site in .NET core - for each separate page of the site (home, recipe input, recipe search etc) do I need to create a new file in the Views folder or can I add a (for example) 'Input.cshtml', 'Search.cshtml' files into the Home view folder which already exists and then add then add the new actions within the HomeController page? Does this make sense? thanks!

**Comment by Nhawdge** [comment]
Yes, that should work, give it a shot!
